backend/inspirehep/records/receivers.py

```diff
@@ -27,13 +25,9 @@ def index_after_commit(sender, changes):
     has been really committed to the DB.
     """
     for model_instance, change in changes:
-        LOGGER.debug("index_after_commit hook")
         if isinstance(model_instance, RecordMetadata):
-            LOGGER.debug(
-                "Model instance (%s) is correct. Processing...", model_instance.id
-            )
             if change in ("insert", "update", "delete"):
-                LOGGER.debug("Change type is %r.", change)
+                LOGGER.debug(f"{change} RecordMetadata")
```

Comment:
It would be helpful to log for which record we have this change from the model_instance